pkg/apis/awsprovider/v1beta1/conversion.go

```diff
@@ -0,0 +1,14 @@
+package v1beta1
+
+import "sigs.k8s.io/cluster-api-provider-aws/pkg/actuators/machine"
+
+var (
+	// RawExtensionFromProviderSpec deprecated: Use machine.RawExtensionFromProviderSpec instead.
```

Comment:
I'm not sure this is the right format, there's a particular format in Go that marks it deprecated and causes compiler warnings, can we check that this happens please?

IIRC it has to be on a separate line
```suggestion
	// RawExtensionFromProviderSpec creates a raw version of the given providerSpec.
	// Deprecated: Use machine.RawExtensionFromProviderSpec instead.
```

Reply:
thanks, I've just pushed it